straxen/contexts.py

```diff
@@ -26,6 +26,8 @@
         straxen.EnergyEstimates,
         straxen.EventInfoDouble,
         straxen.DistinctChannels,
+        straxen.PeakPositionsMLP,
```

Comment:
Later we might rethink whether it is not necessary to list all plugins. Maybe they are already registered earlier.